charts/__tests__/ChartUrl.ts

```diff
@@ -0,0 +1,176 @@
+#! /usr/bin/env yarn jest
+
+import { ChartConfigProps } from "charts/ChartConfig"
+import { Time, TimeBoundValue, TimeBound } from "charts/TimeBounds"
+import { createConfig } from "test/utils"
+
+import { ChartUrl, ChartQueryParams } from "../ChartUrl"
+import { MapConfigProps } from "charts/MapConfig"
+
+type TimeDomain = [Time, Time]
```

Comment:
Good idea – I've now created `TimeBounds`.

These types are very confusing because I gave them a bad name (I couldn't come up with a better one) and because they are the same TypeScript type – `number`. 

`Time` is supposed to represent a number with a finite value, while `TimeBound` can contain `Infinity`. But I couldn't figure out if there is a way to enforce that a bit better...